### DISCUSSION TITLE
Custom naming of artwork

### DISCUSSION THREAD

**Discussion by Sledgehamma** [jellyfin/jellyfin#7969] [General]
I would love to see custom naming of artwork for TV shows and Movies. 
Right now for example Jellyfin does this for movies:
- backdrop
- folder
- logo

One issue is that the home theater App of my Zidoo media player does not recognize “logo” but only “clearlogo”. While batch editing would work, the problem is that I need to do it every time I add a new Tv show or movie. 

On the other hand Emby does this for the same artwork, which I would prefer:
- fanart 
- poster
- clearlogo

I also remember (not checked recently) that Jellyfin did “poster” and “fanart” for TV shows. So it would be great to have the same naming across different media. 

However, I recognize  that  there might not be “one size fits all approach” so that’s why I think it would be best to have the option for “default naming” of the different kinds of artwork.

**Comment by Sledgehamma** [comment]
TinyMediaManager has an extensive option set for naming. It would be really awesome to have something similar for Jellyfin :) 
<a href='https://postimg.cc/TKgjH0Xh' target='_blank'><img src='https://i.postimg.cc/TKgjH0Xh/Screenshot-2022-06-30-at-17-24-27.png' border='0' alt='Screenshot-2022-06-30-at-17-24-27'/></a>
<img width="690" alt="Screenshot 2022-06-30 at 17 24 27" src="https://user-images.githubusercontent.com/19309778/176716063-bd38e02f-dc0b-4196-8abe-c996f0eba33e.png">

**Comment by Damncold** [comment]
or just use a wildcard-names to detect the right image.
```*logo*```
The same for Covers in the music library, would be nice!
